td.vue/src/views/ImportModel.vue

```diff
@@ -62,6 +62,10 @@ import openThreatModel from '@/service/openThreatModel.js';
 import TdFormButton from '@/components/FormButton.vue';
 import tmActions from '@/store/actions/threatmodel.js';
 
+import { THREATMODEL_UPDATE } from '@/store/actions/threatmodel.js';
```

Comment:
THREATMODEL_UPDATE is not used here, so no need for this line
the update store action is used via tmActions.update